src/include/api/vamana_index.h

```diff
@@ -69,7 +70,6 @@ class IndexVamana {
   IndexVamana(IndexVamana&&) = default;
   IndexVamana& operator=(const IndexVamana&) = delete;
   IndexVamana& operator=(IndexVamana&&) = default;
-
```

Comment:
(Minor) There are a good amount of newline deletions / additions in this PR - are they on purpose? If so, that's fine with me, but if they are accident's then let's revert please.

Reply:
I think the pre formatter handles them?

Reply:
No, these line additions and removals are not automatically done, else CI wouldn't be passing on any other PRs..